### PR TITLE
Remove manual cloning

### PR DESCRIPTION
**Pull request by webrouse**
Part of: https://keboola.atlassian.net/browse/COM-1211

### CODE REVIEW

File: internal/pkg/remote/manager.go [hunk 1]
```diff
@@ -144,7 +145,7 @@ func (u *UnitOfWork) loadObject(object model.Object) (model.ObjectState, error)
 	}
 
 	// Set remote state
-	internalObject := object.Clone()
+	internalObject := deepcopy.Copy(object).(model.Object)
```
Comment (webrouse) [inline internal/pkg/remote/manager.go:148]: Uz nepotrebujeme manualne pisat `Clone` metody.
Tu je `remote load`.

File: internal/pkg/remote/manager.go [hunk 2]
```diff
@@ -165,7 +166,7 @@ func (u *UnitOfWork) SaveObject(objectState model.ObjectState, object model.Obje
 	}
 
 	// Invoke mapper
-	apiObject := object.Clone()
+	apiObject := deepcopy.Copy(object).(model.Object)
```
Comment (webrouse) [inline internal/pkg/remote/manager.go:169]: `remote save`.

**Review by webrouse — COMMENTED** [review]
konecne PR, kde sa kod maze :_

**Review by JakubMatejka — APPROVED** [review]
Krása. 😃

**Pull request merged**